internal/servicedeployer/compose.go

```diff
@@ -25,6 +25,9 @@ type DockerComposeServiceDeployer struct {
 	profile  *profile.Profile
 	ymlPaths []string
 	variant  ServiceVariant
+
+	runTeardown  bool
```

Comment:
Nit. In other places we use a different casing, make it consistent.
```suggestion
	runTearDown  bool
```